src/30_Advanced/Embedding_videos_in_amp-accordion.html

```diff
@@ -0,0 +1,176 @@
+<!---{
+    "experiment": true,
+    "component": "amp-accordion"
+  }--->
+<!--
+  #### Introduction
+
+  While it's possible to [integrate videos](https://ampbyexample.com/advanced/integrating_videos_in_amp_an_overview/) in AMP HTML files, when having different videos that you want to display to users, but you want to give the freedom to the user to decide if 
```

Comment:
What is the use case for videos in amp-carousel? Can you explain it a bit more? Are there any challenges that make this sample valuable?